specification/cdn/resource-manager/Microsoft.Cdn/preview/2022-01-01-preview/afdx.json

```diff
@@ -0,0 +1,6282 @@
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "A localized string describing the resource name."
+        }
+      },
+      "description": "The usage names."
+    },
+    "Usage": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource identifier."
+        },
+        "unit": {
+          "type": "string",
+          "description": "An enum describing the unit of measurement.",
+          "enum": [
+            "Count"
+          ],
+          "x-ms-enum": {
+            "name": "UsageUnit",
+            "modelAsString": true
+          }
+        },
+        "currentValue": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The current value of the usage."
+        },
+        "limit": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The limit of usage."
+        },
+        "name": {
+          "$ref": "#/definitions/UsageName",
+          "description": "The name of the type of usage."
+        }
+      },
+      "required": [
+        "unit",
+        "currentValue",
+        "limit",
+        "name"
+      ],
+      "description": "Describes resource usage."
+    },
+    "UsagesListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          },
+          "description": "The list of resource usages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of results."
+        }
+      },
+      "description": "The list usages operation response."
+    },
+    "ActivatedResourceReference": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        },
+        "isActive": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Whether the resource is active or inactive"
+        }
+      },
+      "description": "Reference to another resource along with its state.",
+      "type": "object",
+      "x-ms-azure-resource": true
+    },
+    "Policy": {
+      "description": "A policy is a logical grouping of policies",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AFDStateProperties"
+        },
+        {
+          "$ref": "./cdn.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyProperties"
+        }
+      }
+    },
+    "PolicyProperties": {
+      "description": "The JSON object that contains the properties of the Policy to create.",
+      "type": "object",
+      "required": [
+        "policyData",
+        "serviceType",
+        "policyVersion"
+      ],
+      "properties": {
+        "policyData": {
```

Comment:
what are policies and what is an example "policyData" string?  The example files just contain empty strings

Reply:
It is a string

Reply:
What does a policy do and what does this "data" look like?  Please provide an example of what someone might set here and what the outcome would be